.CMake/alg_support.cmake

```diff
@@ -176,26 +176,8 @@ endif()
 
 option(OQS_ENABLE_KEM_HQC "Enable hqc algorithm family" ON)
 cmake_dependent_option(OQS_ENABLE_KEM_hqc_128 "" ON "OQS_ENABLE_KEM_HQC" OFF)
-if(CMAKE_SYSTEM_NAME MATCHES "Linux|Darwin")
```

Comment:
Do I understand this right meaning that HQC doesn't contain AVX2 optimizations any more? Note to (at least) self: Don't be surprised about new HQC performance numbers then.

Reply:
Makes sense. Would it be reasonable then to create an issue for integrating that HQC AVX2 code in the future?

Reply:
Yes, I'll do that right now.